packages/babel-plugin-emotion/src/index.js

```diff
@@ -228,6 +228,34 @@ export function replaceCssObjectCallExpression(path, identifier, t) {
 
 const visited = Symbol('visited')
 
+const parseImports = (node, currentNames) => {
+  const names = Object.assign({}, currentNames)
+  const imports = node.specifiers.filter(
+    specifier =>
+      specifier.type === 'ImportDefaultSpecifier' ||
+      specifier.type === 'ImportSpecifier'
+  )
+
+  imports.forEach(singleImport => {
+    if (singleImport.imported) {
+      // Is helper method
+      names[singleImport.imported.name] = singleImport.local.name
+    } else {
+      // Is default import
+      names.default = singleImport.local.name
+    }
+  })
```

Comment:
This can be done with `state.file.metadata.modules.imports` in the `Program` enter visitor.

http://astexplorer.net/#/gist/39f0d575f432d2a93f29c0aebe34956c/36992fb4d485246b651b79d3579756ef016c9c5c

Reply:
Thanks for the suggestion. Was able to remove `parseImports` and the `ImportDeclaration` visitor.